### PR TITLE
Retrieve sleep data from Google Fit

### PR DESCRIPTION
**Pull request by mschristo**
I tried to add a way to get the sleep records from Google Fit. The return object contains start-end time of each segment and the type (sleep_stage) of this segment (https://developers.google.com/android/reference/com/google/android/gms/fitness/data/SleepStages)

### CODE REVIEW

**Review by dariosalvi78 — CHANGES_REQUESTED** [review]
this is great, thanks!
Instead of returning the integer as it is, can you please return a string using the following map:

SLEEP  -> sleep
AWAKE  -> sleep.awake
OUT_OF_BED -> sleep.outOfBed
SLEEP_DEEP -> sleep.deep
SLEEP_LIGHT -> sleep.light
SLEEP_REM -> sleep.rem

this is compatible with the [activity recognition categories ](https://github.com/dariosalvi78/cordova-plugin-health/blob/master/activities_map.md)

File: README.md [hunk 1]
```diff
@@ -174,6 +174,7 @@ Example values:
 | mindfulness     | 1800 <br/>**Notes**: only available on iOS |
 | nutrition      | { item: "cheese", meal_type: "lunch", brand_name: "McDonald's", nutrients: { nutrition.fat.saturated: 11.5, nutrition.calories: 233.1 } } <br/>**Note**: the `brand_name` property is only available on iOS |
 | nutrition.X    | 12.4                              |
+| sleep       | 2 <br />**Notes**: recognized sleep stages and their mappings in Google Fit can be found [here](https://developers.google.com/android/reference/com/google/android/gms/fitness/data/SleepStages#constant-summary) |
```
Comment (dariosalvi78) [inline README.md:177]: specify the categories here:

sleep, sleep.awake, sleep.outOfBed, sleep.deep, sleep.light, sleep.rem

File: src/android/HealthPlugin.java [hunk 2]
```diff
@@ -744,7 +745,11 @@ else if (mealt == Field.MEAL_TYPE_SNACK)
           }
           obj.put("value", bpobj);
           obj.put("unit", "mmHg");
-        }
+        }  else if (dt.equals(DataType.TYPE_SLEEP_SEGMENT)) {
+          int sleepSegmentType = datapoint.getValue(Field.FIELD_SLEEP_SEGMENT_TYPE).asInt();
```
Comment (dariosalvi78) [inline src/android/HealthPlugin.java:749]: map the integer to the following strings:
SLEEP  -> sleep
AWAKE  -> sleep.awake
OUT_OF_BED -> sleep.outOfBed
SLEEP_DEEP -> sleep.deep
SLEEP_LIGHT -> sleep.light
SLEEP_REM -> sleep.rem

**Review by dariosalvi78 — CHANGES_REQUESTED** [review]

File: src/android/HealthPlugin.java (hunk 2 above)
Comment (mschristo) [reply src/android/HealthPlugin.java]: Hi! 

Thank you for your response.

I just added a commit with your proposed change.

**Pull request merged**